guide/payments/privacy.md

```diff
@@ -0,0 +1,139 @@
+> * Word gets around. A burglar raids your home. Kidnappers capture your children and know exactly how much to demand in ransom.
+
+-->
+
+
+**Do's**
+
+- Show a new unused address when the receive screen is entered.
+- Make it easy to generate as many addresses as the receiver needs at a time.
+- Before broadcasting a transaction, inform senders if an address is already used.
+
+**Don'ts**
+
+- Make it easy to reuse an address.
+
+### Keep track of who knows about an address
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/address-labeling.jpg"
+   retina = "/assets/images/guide/payments/privacy/address-labeling@2x.jpg"
+   alt-text = "A list of addresses with tags. Two are red, two are purple and one is blue"
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+When [receiving bitcoin]({{ "/guide/payments/receive/" | relative_url }}) we can [add additional details to a payment]({{ "/guide/payments/receive/#inputting-additional-payment-details" | relative_url }}). Not only does this help to contextualize payments, but it also enables preventative measures for preserving privacy. Labeling addresses with the counterparty's name involved in each transaction can inform decisions for which [UTXOs]({{ "/guide/glossary/#unspent-transaction-output-utxo" | relative_url }}) are selected in future transaction inputs.
+
+To make this UTXO selection easier, some interfaces make it possible to select/filter UTXOs that cluster under one label.
+
+_Check out the glossary to learn more about [how coin control works]({{ "/guide/glossary/#coin-control" | relative_url }})._
+
+</div>
+
+### Increase anonymity by collaborating with others
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/coin-join.jpg"
+   retina = "/assets/images/guide/payments/privacy/coin-join@2x.jpg"
+   alt-text = "Example CoinJoin transaction with 3 participants with 1 input each, which get broken up into 5 equal outputs."
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+The "common input ownership" heuristic[^4] assumes in a transaction, it is likely that all inputs belong to the same owner. [CoinJoins]({{ "/guide/glossary/#coinjoin" | relative_url }}) is a technique where multiple participants collaborate on a transaction to break this heuristic. It becomes more difficult (although not impossible) to determine the individuals who own the outputs are since all the outputs typically have the same denomination. With CoinJoins, as with any other anonymity network, a large and diverse group of participants provides the best protection.
+
+Users still have to be mindful of how the UTXOs they received from the CoinJoin are spent. For instance, spending them together in a single transaction would unravel the anonymity gains from participating in the CoinJoin.
+
+</div>
+
+### Design with privacy in mind
```

Comment:
This page's context is a bit confusing - is it about privacy in the context of payments (as it part of the payments chapter) or Bitcoin privacy application wide? I think the context needs to applied more throughout. 

It makes more sense to intertwine our principles into all the pages content rather than have separate pages that cover a principle. e.g. instead of a 'privacy payments' page we integrate this privacy content into the send/receive pages.

Reply:
The send and receive pages should definitely mention/implement/not contradict the content here, but I still believe it reads well having a dedicated page to this subject.